packages/core/src/shared/filesystemUtilities.ts

```diff
@@ -245,7 +245,7 @@ export async function setDefaultDownloadPath(downloadPath: string) {
             await globals.globalState.update('aws.downloadPath', path.dirname(downloadPath))
         }
     } catch (err) {
-        getLogger().error('Error while setting "aws.downloadPath"', err as Error)
+        getLogger().error('Error while setting "aws.downloadPath" \n %O', err as Error)
```

Comment:
nit: it's usually quite readable to start the object at the end of the line, it will look like:

```
error: ... {
  ...
}
```
```suggestion
        getLogger().error('Error while setting "aws.downloadPath": %O', err as Error)
```